src/components/DiagnosticsBody.vue

```diff
@@ -97,7 +97,7 @@ export default Vue.extend({
     <div class="status">
       <div class="result-info">
         <div class="item-results">
-          <span class="icon icon-dot text-error" />{{ numFailed }} failed ({{ numMuted }} muted)
+          <span class="icon icon-dot text-error" />{{ numFailed }} failed, {{ numMuted }} muted
```

Comment:
```suggestion
          <span class="icon icon-dot text-error" />{{ numFailed }} failed plus {{ numMuted }} muted
```
Also, maybe only only show the second part if `numMuted > 0`?

Reply:
Yeah, I'm looking at this too right now, and the "failed" number is really redundant with the number in the badge icon, so I wonder if we should repeat it at all.

Anyways, for now let's go with the format Mark suggested:
* `1 failed plus 1 muted` (I suggested "and" instead of "plus", and still prefer it, but not by much)
* `1 failed` (if number of muted it 0)

Reply:
We can do this, but I'm not thrilled by the inconsistency.
The user has to do less processing if we always use the same format, meaning:

```
3 failed, 0 muted
0 failed, 2 muted
0 failed, 0 muted
```

It's the same as the output from a test suite. Almost every test framework I'm familiar with
emits 0-values, even for something like `n passed, 0 failed, 0 skipped, 0 not ready.`

I'm fine with the disappearing badge icon, because then there's no red number suggesting the user click on the tab.

Reply:
Ok, let's keep the muted number in there, even when it is 0. Otherwise we should really also hide the "Show muted" toggle as well.

But please use "1 failed and 0 muted" instead of "1 failed, 0 muted"

Reply:
Here are the screen shots (with deliberately broken links and a k8s context):

3-failed-0-muted.png:
![3-failed-0-muted](https://user-images.githubusercontent.com/108392/191628111-db750cae-c735-4f8c-b756-2bb2463ed59d.png)

2-failed-1-muted.png:
![2-failed-1-muted](https://user-images.githubusercontent.com/108392/191628130-0160611e-2a39-4cb9-9623-64721887464f.png)

1-failed-2-muted.png:
![1-failed-2-muted](https://user-images.githubusercontent.com/108392/191628187-0b64b1b3-c0f3-4ce7-8cda-40ce0bc5ffce.png)

0-failed-3-muted-show-muted.png:
![0-failed-3-muted-show-muted](https://user-images.githubusercontent.com/108392/191628209-14f5e73f-3608-40d6-ba61-f411da8e4da4.png)

0-failed-3-muted-hide-muted.png:
![0-failed-3-muted-hide-muted](https://user-images.githubusercontent.com/108392/191628232-5f3e2f4c-b08b-4053-992f-0975b01db8e2.png)